brainframe_qt/api_utils/streaming/zone_status_frame.py

```diff
@@ -38,6 +40,13 @@ class ZoneStatusFrame:
         'frame_metadata': 'ZoneStatusFrameMeta',
     }
 
+    @staticmethod
+    def pixmap_from_numpy_frame(frame: np.ndarray) -> QPixmap:
+        height, width, channel = frame.shape
+        bytes_per_line = width * 3
+        image = QImage(frame.data, width, height, bytes_per_line, QImage.Format_RGB888)
```

Comment:
Aren't our Numpy frames always in BGR format? I assume it would be obvious if this didn't work so I'm sure you're right, I'm just surprised.

Reply:
Oh, alright.

Reply:
There's a BGR format, but it's in a new version of Qt we're not using yet